src/main/java/com/puppycrawl/tools/checkstyle/xpath/XpathQueryGenerator.java

```diff
@@ -75,18 +80,34 @@
     private final int lineNumber;
     /** The column number of the element for which the query should be generated. */
     private final int columnNumber;
+    /** The {@code FileText} object, representing content of the file. */
+    private final FileText fileText;
+    /** The distance between tab stop position. */
+    private final int tabWidth;
 
     /**
      * Creates a new {@code XpathQueryGenerator} instance.
      *
      * @param rootAst root ast
      * @param lineNumber line number of the element for which the query should be generated
      * @param columnNumber column number of the element for which the query should be generated
+     * @param fileName path to the file
+     * @param tabWidth distance between tab stop position
      */
-    public XpathQueryGenerator(DetailAST rootAst, int lineNumber, int columnNumber) {
+    public XpathQueryGenerator(DetailAST rootAst, int lineNumber, int columnNumber,
+                               String fileName, int tabWidth) {
         this.rootAst = rootAst;
         this.lineNumber = lineNumber;
         this.columnNumber = columnNumber;
+        try {
+            fileText = new FileText(new File(fileName),
+                    StandardCharsets.UTF_8.name());
+        }
+        catch (IOException ex) {
+            throw new IllegalStateException("I/O exception occurred when reading file :"
```

Comment:
`reading file :"` => `reading file: "` (space after `:` not before)

Reply:
done